client/js/components/common/movable_words/word_chain.jsx

```diff
@@ -4,15 +4,21 @@ import ReactDOM            from 'react-dom';
 import { WordDropZone }    from '../drop_zones';
 import DraggableGroupWord  from './draggable_group_word';
 
-const beginWrap = <svg xmlns="http://www.w3.org/2000/svg" width="48" height="48" viewBox="0 0 48 48">
+export const beginWrap = <svg xmlns="http://www.w3.org/2000/svg" width="48" height="48" viewBox="0 0 48 48">
```

Comment:
We should move these into a constants file and then each component can import that file.